tools/mason/MasonNew.chpl

```diff
@@ -35,8 +35,12 @@ proc masonNew(args) throws {
     } else {
       var vcs = true;
       var show = false;
+      var isModuleName = false;
+      var moduleName = '';
       var name = '';
+      var i = 0;
```

Comment:
Would it make sense to start this counter at 2 so we don't have to offset elsewhere?